README.md

```diff
@@ -95,6 +106,12 @@ The rules created are:
 
 ### Interacting with your cluster
 
+If you are using kind, config the `KUBECONFIG` first before using kubectl:
+
+```bash
+export KUBECONFIG="$(kind get kubeconfig-path --name="clusterapi")"
```

Comment:
I didn't use kind before. kind don't have --kubeconfig param?

Reply:
Yes, we have to use the command `kind get kubeconfig-path --name="clusterapi"` to get configuration path, so that's why I want to highlight this here. ;-)